tests/hash-backend-integration/src/tests/graph/knowledge/primitive/entity.test.ts

```diff
@@ -63,7 +64,8 @@ describe("Entity CRU", () => {
   let testUser: User;
   let testUser2: User;
   let entityType: EntityTypeWithMetadata;
-  let textDataType: DataTypeWithMetadata;
+  const textDataTypeId =
+    "https://blockprotocol.org/@blockprotocol/types/data-type/text/v/1" as VersionedUrl;
```

Comment:
Can we put this somewhere shared? maybe in `../../util` since these files are already importing from it

Reply:
Applied in 87ece23e4022595008654ae4c173ccf38f3ac17c